test/combos/unmarshaler/thetest.proto

```diff
@@ -624,3 +624,10 @@ message Node {
 	repeated Node Children = 2;
 }
 
+message NonByteCustomType {
+  optional ProtoType Field1 = 1 [(gogoproto.customtype) = "GoType"];
```

Comment:
Just thought of this the other day, but can we also have structs which use nullable=false and repeated custom types, similar to NidOptCustom, NinOptCustom, NidRepCustom and NinRepCustom